src/catkin_pkg/templates/CMakeLists.txt.in

```diff
@@ -158,11 +158,18 @@ include_directories(
 #   DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
 # )
 
-## Mark executables and/or libraries for installation
-# install(TARGETS ${PROJECT_NAME} ${PROJECT_NAME}_node
+## Mark executables for installation
+## See http://docs.ros.org/melodic/api/catkin/html/howto/format1/building_executables.html
```

Comment:
@seanyen: did you link to the format 1 pages intentionally?

Reply:
No, I didn't notice I put format 1 pages here. I will send another pull request to update it to format 2.